src/charm.py

```diff
@@ -69,7 +76,11 @@ def _configure_sdcore_nms(self, event: EventBase) -> None:
             self.unit.status = WaitingStatus("Waiting for container to be ready")
             event.defer()
             return
-        for required_relation in [FIVEG_N4_RELATION_NAME, SDCORE_MANAGEMENT_RELATION_NAME]:
+        for required_relation in [
+            FIVEG_N4_RELATION_NAME,
+            SDCORE_MANAGEMENT_RELATION_NAME,
+            GNB_IDENTITY_RELATION_NAME,
+        ]:
```

Comment:
N4 and gNB should not be required relations. We shouldn't have to deploy a gNodeB to use NMS.